src/_pytest/main.py

```diff
@@ -175,6 +178,16 @@ def pytest_addoption(parser):
     )
 
 
+def validate_basetemp(path):
+    # GH 7119
+    cwd = pathlib.Path.cwd()
```

Comment:
I'd go for `os.getcwd()` here -- pathlib doesn't add much in this case as written. (But see next comment where `pathlib` might in fact be useful).